README.md

```diff
@@ -30,18 +31,18 @@ As it is presented the information for the contacts is only stored in memory and
 
 ## The tasks
 
-1. Our customers requested a new feature related to the coronavirus restriction. They have to clean each rental unit after their guest occupies it. To do it we should block additional X days after each booking. Block means to not allow to have any booking for the same unit during this time. The number of days will be defined by the host.
+1. Our customers requested a new feature related to the coronavirus restriction. They have to clean each rental unit after their guest occupies it. To achieve this we should block additional **X days** after each booking. Block means to not allow to have any booking for the same unit during this time. The number of days will be defined by the host.
 
 E.g.: 1 day preparation time:
-For any booking without other booking before nor after, a day is blocked after it for the unit where the booking occurs.
+For any booking without another booking before nor after, an extra day is blocked for the unit where the existing booking occurs.
 
-One booking always occupies only one unit. The rental can have many units. So a rental with 3 units is fully occupated for the selected night when it has (3 bookings) OR (2 bookings and 1 preparation time period) OR (1 booking and 2 preparation time periods) OR (3 preparation time periods).
+One booking always occupies only one unit. The rental can have many units. So a rental with 3 units is fully occupied for the selected night when it has (3 bookings) OR (2 bookings and 1 preparation time period) OR (1 booking and 2 preparation time periods) OR (3 preparation time periods).
 
 The contracts with API consumers are already defined. You have to extend existing endpoints to fulfill new definitions and provide the requested functionality.
 
 A new `Unit` property is added to the `Booking` calendar model and to the new `PreparationTime` calendar model. This property is a number that references one of the available units in the Rental. For example, if the Rental has a total of 3 units, the Unit property can accept the values 1, 2,  or 3. It is irrelevant whether the Booking occupies unit 1, 2 or 3, as long as its associated PreparationTime occupies the same Unit number as the Booking.
 
-If the calendar model contains `Booking` with `Unit` 2 then it means that `Booking` is associated to `Unit` number 2. It does NOT mean that the booking occupies two units.
+If the calendar model contains `Booking` with `Unit` 2 then it means that `Booking` is associated to or booked at `Unit` number 2. **It does NOT mean that the booking occupies two units**.
```

Comment:
```suggestion
If the calendar model contains `Booking` with `Unit` 2 then it means that `Booking` is booked at `Unit` number 2. **It does NOT mean that the booking occupies two units**.
```

Because I think associated == booked in this case